src/Swarm/TUI/Model.hs

```diff
@@ -235,6 +234,12 @@ data Name
     RobotPanel
   | -- | The info panel on the bottom left.
     InfoPanel
+  deriving (Eq, Ord, Show, Read, Bounded, Enum)
+
+-- | 'Name' represents names to uniquely identify various components
+--   of the UI, such as forms, panels, caches, extents, and lists.
+data Name
+  = FocusablePanel PanelFocusable
```

Comment:
Why are the names of these two things backwards from each other?  There is no technical reason they can't have the same name and I would never be able to keep straight which is which.

Reply:
Ah you're right, since types and data constructors are different namespaces. I've consolidated on `FocusablePanel`.